axon-server-connector/src/main/java/org/axonframework/axonserver/connector/heartbeat/HeartbeatMonitor.java

```diff
@@ -0,0 +1,59 @@
+package org.axonframework.axonserver.connector.heartbeat;
+
+import org.axonframework.axonserver.connector.AxonServerConnectionManager;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Verifies if the connection is still alive, and react if it is not.
+ *
+ * @author Sara Pellegrini
+ * @since 4.2
+ */
+public class HeartbeatMonitor {
+
+    private final Logger log = LoggerFactory.getLogger(HeartbeatMonitor.class);
```

Comment:
The regular naming for a `Logger` instance is `logger`, and we always make them `private static final` in the framework.